src/Common/src/System/Drawing/KnownColorTable.cs

```diff
@@ -56,9 +52,6 @@ private static void InitColorTable()
         {
             int[] values = new int[(unchecked((int)KnownColor.MenuHighlight)) + 1];
 
-#if FEATURE_SYSTEM_EVENTS
-            SystemEvents.UserPreferenceChanging += new UserPreferenceChangingEventHandler(OnUserPreferenceChanging);
-#endif
```

Comment:
I would prefer to keep the code here until we decide we can't or don't want to ever reactivate it. Let's discuss it over at #21319